tests/acceptance.suite.yml

```
@@ -0,0 +1,18 @@
+# Codeception Test Suite Configuration
+#
+# Suite for acceptance tests.
+# Perform tests in browser using the WebDriver or PhpBrowser.
+# If you need both WebDriver and PHPBrowser tests - create a separate suite.
+
+actor: AcceptanceTester
+modules:
+  enabled:
+    - WebDriver:
+        url: https://mautic.ddev.site
```

The url does not seem to work when using gitpod. Since the tests are also run inside of ddev, you can use the following url:

```suggestion
        url: "https://web:443"
```

Other that that, the tests worked like they should 👍

---

Thanks. I have updated it.